### ISSUE TITLE
missing loss_fn definition in Listing 2.9 and "chp2 embeddings.ipynb"

### ISSUE THREAD

**Issue by jhconning** [keitabroadwater/gnns_in_action#9]
I'm loving the book, but I have found a couple missing elements in the code (one already identified in #3):

Listing 2.9 of the book (and in several places of the chp2 notebook) you use:

`loss = loss_fn(out_masked, train_labels)`

But a loss function is not defined here or in later instances in the same notebook where it is used.  So the code does not run.

Given the classification  task and the log_softmax output, the following seems to work:

`loss_fn = torch.nn.NLLLoss()`
